tests/components/axis/test_config_flow.py

```diff
@@ -36,6 +36,13 @@
 from tests.common import MockConfigEntry
 
 
+@pytest.fixture(name="mock_config_entry")
+async def mock_config_entry_fixture(prepare_config_entry):
```

Comment:
Maybe not compulsory, but I find that it helps to make it "autouse=True" in `test_config_flow.py`.
Then if tests require a "real" setup then I split or move them to `test_init.py`

Reply:
yes - that's what I prefer also
what I mean about splitting is that in my case I had a single test case for "trigger reauth + handle reauth"
Now "triggering reauth" is tested in `test_init`, and "handling reauth" stays in `test_config_flow`

Reply:
Looked through the tests and this change would conflict with the options flow tests.